### DISCUSSION TITLE
README.md Plex: Hardware Acceleration Support

### DISCUSSION THREAD

**Discussion by nichtmax** [tteck/Proxmox#609] [Ideas]
The Hardware Acceleration Support mentioned here is for Intel CPUs only, it took me quite some time to figure that out. 

Plex supports Nvidia GPUs too, for these a [whole different setup is needed](https://jocke.no/2022/02/23/plex-gpu-transcoding-in-docker-on-lxc-on-proxmox/). I could try to do a PR if you're interested?

Cheers,
Moritz

**Comment by nichtmax** [comment]
See https://github.com/tteck/Proxmox/discussions/579

**Comment by tteck** [comment]
PR's are welcomed. Please use the `pull-requests` branch.
